extensions/windows-event-log/tests/CWELCustomProviderTests.cpp

```diff
@@ -172,9 +172,10 @@ TEST_CASE("ConsumeWindowsEventLog prints events in JSON::Flattened correctly cus
     {
       "Name": ")" + CUSTOM_PROVIDER_NAME + R"(",
       "Channel": ")" + CUSTOM_CHANNEL /* Channel is not overwritten by data named "Channel" */ + R"(",
-      "EventData": )" + EVENT_DATA_JSON /* EventData is not discarded */ + R"(,
-      "param1": "Actual event",
-      "param2": "Second"
+      "EventData.param1.Data": "Actual event",
+      "EventData.param2.Data": "Second",
+      "EventData.Channel.Data": "Third",
+      "EventData.Binary": "0901"
```

Comment:
I think we should drop the type info from here, if there is a name. It doesn't seem to add any value.
```suggestion
      "EventData.param1": "Actual event",
      "EventData.param2": "Second",
      "EventData.Channel": "Third",
      "EventData.Binary": "0901"
```